
pan_tilt_topic: 'pan_tilt_controller/command'
joy_sub_topic: 'joy'
joy_pan_axis: 6
joy_tilt_axis: 7
increament_pan: 0.1
increament_tilt: 0.1
zero_button: 3
joy_deadman_button: 7

elevator_increment: 0.1
elevator_up_button: 4
elevator_down_button: 0 

drive_joy_teleop_linear_axis: 3
drive_joy_teleop_angular_axis: 2
drive_joy_teleop_deadman_button: 7
scale: 1.0
